### PR TITLE
fix: Filters are not applied unless I'm on the All Conversations screen

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/product/issues/654

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
https://www.loom.com/share/8ab197c440ac4f969628c6e212f649b3

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nithindavid — APPROVED** [review]

**Comment by nithindavid** [comment]
@pranavrajs If we are applying the filters on inboxes or label filter pages, the actual functionality expects the filter to have  inbox or label pre filled? Right now it's all conversations?

**Comment by iamsivin** [comment]
> @iamsivin One thing I noticed is the header still shows "Unattended" or "" even after I apply the filters. As @nithindavid's previous comment, this would be a misleading title. Can we make sure that if the filters are applied, it will show Conversations instead of the name of the view?

@pranavrajs Done https://github.com/chatwoot/chatwoot/pull/6006/commits/aec5038b347ca10dccf93190fac956332709df85

File: app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue [hunk 1]
```diff
@@ -112,6 +112,7 @@ $label-badge-size: var(--space-slab);
   padding: var(--space-smaller) var(--space-smaller);
   margin: var(--space-smaller) 0;
   text-align: left;
+  line-height: 1.2;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue:115]: Will fix this issue
Screenshot 2022-12-05 at 10 39 33 PM

File: app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue [hunk 2]
```diff
@@ -135,8 +136,6 @@ $label-badge-size: var(--space-slab);
 
 .menu-label {
   flex-grow: 1;
-  display: inline-flex;
-  align-items: center;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue:139]: Will fix this overflow hidden issue.
Screenshot 2022-12-05 at 10 39 04 PM

File: app/javascript/dashboard/components/ChatList.vue [hunk 3]
```diff
@@ -345,23 +345,26 @@ export default {
...
+        if (this.conversationType === 'mention') {
+          return this.$t('CHAT_LIST.MENTION_HEADING');
+        }
+        if (this.conversationType === 'unattended') {
+          return this.$t('CHAT_LIST.UNATTENDED_HEADING');
+        }
+        if (this.hasActiveFolders) {
+          return this.activeFolder.name;
+        }
+        return this.$t('CHAT_LIST.TAB_HEADING');
```
[the remark this answers is not in the document]
Comment (iamsivin) [reply app/javascript/dashboard/components/ChatList.vue]: Done https://github.com/chatwoot/chatwoot/pull/6006/commits/df2f74d154d8cf5b48506f20128194abd0a8bc87

**Review by nithindavid — APPROVED** [review]

**Pull request merged**